### PR TITLE
Fixed some Pydantic case parsing model types and default values

### PR DESCRIPTION
**Pull request by northwestwitch**
Some model types in pydantic parsing are incorrect or could have better (or none) values

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
</details>

**How to test**:
1. Load a demo case with and without the modified params in the config file and check that the created case document is correct

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by DN
- [x] tests executed by CR

### CODE REVIEW

File: scout/parse/models.py [hunk 1]
```diff
@@ -281,15 +281,15 @@ class ScoutLoadConfig(BaseModel):
...
+    madeline_info: Optional[str] = Field(None, alias="madeline")  #!
     multiqc: Optional[str] = None
     owner: str = None
     peddy_ped: Optional[str] = None
-    peddy_ped_check: Optional[str] = Field("", alias="peddy_check")
-    peddy_sex_check: Optional[str] = Field("", alias="peddy_sex")
+    peddy_ped_check: Optional[str] = Field(None, alias="peddy_check")
+    peddy_sex_check: Optional[str] = Field(None, alias="peddy_sex")
     phenotype_terms: Optional[List[str]] = None
     rank_model_version: Optional[str] = ""
```
Comment (dnil) [inline scout/parse/models.py:294]: What about this and the sv_rank_model_version? None seems more prudent, but I haven't checked in the templates/controllers.

File: .github/workflows/vulture.yml [hunk 2]
```diff
@@ -19,5 +19,5 @@ jobs:
         uses: anaynayak/python-vulture-action@v1.0
         id: vulture
         with:
-          vulture-args: --min-confidence 80 ${{steps.files.outputs.all}}
+          vulture-args: --min-confidence 80 --ignore-names cls,args ${{steps.files.outputs.all}}
```
Comment (dnil) [inline .github/workflows/vulture.yml:22]: Sure, that saves a lot of #noqa: F841!

File: scout/parse/models.py (hunk 1 above)
Comment (northwestwitch) [reply scout/parse/models.py:294]: I was thinking the same, I'll check!

Comment (northwestwitch) [reply scout/parse/models.py]: After setting them to None, and trying to load demo case, the values are still correctly parsed:

image

Comment (northwestwitch) [reply scout/parse/models.py]: If case config.yaml doesn't contain any rank_model_version or sv_rank_model_version, then the created document case doesn't contain these keys, which I think is fine.

**Pull request merged**